src/tools/auth0/handlers/actions.ts

```diff
@@ -92,29 +93,34 @@ export function isMarketplaceAction(action: Action): boolean {
 }
 
 export default class ActionHandler extends DefaultAPIHandler {
-  existing: Action[] | null;
+  existing: GetActions200ResponseActionsInner[] | null;
 
   constructor(options: DefaultAPIHandler) {
     super({
       ...options,
       type: 'actions',
       functions: {
-        create: (action: Action) => this.createAction(action),
+        create: (action: PostActionRequest) => this.createAction(action),
         delete: (action: Action) => this.deleteAction(action),
       },
       stripUpdateFields: ['deployed', 'status'],
     });
   }
 
-  async createAction(action: Action) {
+  async createAction(action: PostActionRequest) {
     // Strip the deployed flag
     const addAction = { ...action };
-    delete addAction.deployed;
-    delete addAction.status;
-    const createdAction = await this.client.actions.create(addAction);
 
+    // TODO: Should we keep this?
+    delete (addAction as any).deployed;
+    delete (addAction as any).status;
```

Comment:
Yes because when you create an action you cannot specify the `deploy` or `status` in the payload. However, this should be accomplished with the `stripCreateFields: ['deployed', 'status'],` in the handler definition above.